Gemfile

```diff
@@ -4,7 +4,7 @@ source 'https://rubygems.org'
 # We don't want to list psych since that updates the bundled version
 gem 'rdoc', '< 6.4'
 
-gem 'kafo', '>= 7.3', '< 8'
+gem 'kafo', github: 'theforeman/kafo'
```

Comment:
This needs updating once Kafo 7.6 is released with the feature (see https://github.com/theforeman/kafo/pull/380), but other than that this is ready for review